anime_downloader/sites/dreamanime.py

```diff
@@ -0,0 +1,69 @@
+
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+from urllib.parse import quote_plus
+import json
+import re
+
+class DreamAnime(Anime, sitename='dreamanime'):
+    """
+    Site: http://dreamanime.fun
+    Config
+    ------
+    version: One of ['subbed', 'dubbed]
+        Selects the version of audio of anime.
+    server: One of ['mp4upload', 'trollvid']
+        Selects the server to download from.
+    """
+
+    sitename='dreamanime'
+    url = f'https://{sitename}.fun/search'
+
+    @classmethod
+    def search(cls, query):
+        results = helpers.get("https://dreamanime.fun/search?term=" + quote_plus(query)).text
```

Comment:
Unnecessary hardcoding of url parameters, search_url is defined on line 20 as url but manually inputted here. Should use url and pass parameters as such helpers.get('https://dreamanime.fun/search', params={'term': 'naruto'}).text and remove import of quote_plus.